cmd/kube-scheduler/app/options/options.go

```diff
@@ -277,6 +277,39 @@ func (o *Options) Config() (*schedulerappconfig.Config, error) {
 
 	c.Client = client
 	c.InformerFactory = informers.NewSharedInformerFactory(client, 0)
+
+	// if the resource provider kubeconfig is not set, default to the local cluster
+	if c.ComponentConfig.ResourceProviderKubeConfig == "" {
+		klog.V(2).Infof("ResourceProvider kubeConfig is not set. default to local cluster client")
+		c.NodeInformers = make(map[string]coreinformers.NodeInformer, 1)
+		c.NodeInformers["rp0"] = c.InformerFactory.Core().V1().Nodes()
+	} else {
+		kubeConfigFiles, existed := genutils.ParseKubeConfigFiles(c.ComponentConfig.ResourceProviderKubeConfig)
+		// TODO: once the perf test env setup is improved so the order of TP, RP cluster is not required
```

Comment:
this comment seems obsoleted to the code below.

Reply:
comment rewrote